Mathlib/Analysis/Calculus/FDeriv/Add.lean

```diff
@@ -670,32 +670,32 @@ theorem DifferentiableAt.sub_const (hf : DifferentiableAt 𝕜 f x) (c : F) :
   (hf.hasFDerivAt.sub_const c).differentiableAt
 #align differentiable_at.sub_const DifferentiableAt.sub_const
 
-@[deprecated DifferentiableAt.sub_iff_left]
+@[deprecated DifferentiableAt.sub_iff_left (since := "2024-07-11")]
 theorem differentiableAt_sub_const_iff (c : F) :
-    DifferentiableAt 𝕜 (fun y => f y - c) x ↔ DifferentiableAt 𝕜 f x := by
-  simp only [sub_eq_add_neg, differentiableAt_add_const_iff]
+    DifferentiableAt 𝕜 (fun y => f y - c) x ↔ DifferentiableAt 𝕜 f x :=
+  DifferentiableAt.sub_iff_left (differentiableAt_const _)
```

Comment:
If you didn't know about this, this is called dot notation and is the entire point of naming lemmas about `FancyProp` like `FancyProp.add`, `FancyProp.neg`, ...
```suggestion
  (differentiableAt_const _).sub_iff_left 
```

Reply:
Thanks. I did know about this but just "copy pasted" without thinking.